modules/OCPP/OCPP.cpp

```diff
@@ -901,6 +901,13 @@ void OCPP::ready() {
         this->evse_ready_cv.wait(lk);
     }
 
+    // if charger information interface is connected, override only these specific properties
+    // which were loaded from configuration file(s)
+    if (!this->r_charger_information.empty()) {
+        auto ci = this->r_charger_information.at(0)->call_get_charger_information();
+        this->charge_point->update_chargepoint_information(ci.vendor, ci.model, ci.serial, ci.firmware_version);
```

Comment:
In 1.6 (and 2.0.1 as well as 2.1) there's also some `modem` information you could transmit in the `BootNotification`, namely `ICCID` and `IMSI`. Maybe we want to set this information as well